### ISSUE TITLE
Calculating a Perpendicular Vector in Basis 2

### ISSUE THREAD

**Issue by umbriquse** [chakravala/Grassmann.jl#91]
Hello, I am a fan of the package.

I ran across an issue when trying to find a perpendicular vector. The issue being that the value returned when calculating everything at once differs from an equivalent, but broken up, calculation. Below is a MWE of the behavior. I don't know if this is something that I am doing, but this seem wrong.
```
using Grassmann
basis"2"
a = -0.7296052180768686v₁ + 0.5774086088345938v₂
b = 0.0v₁ + 2v₂
@show (a ∧ b) * inv(b)
c = (a ∧ b).v[1] * v12
d = inv(b)
@show c * d
```
OUTPUT
```
(a ∧ b) * inv(b) = 0.0v
c * d = -0.7296052180768686v₁ + 0.0v₂
-0.7296052180768686v₁ + 0.0v₂
```
PACKAGE STATUS
```
(Agate) pkg> st Grassmann
      Status `~/git/Agate.jl/Project.toml`
  [4df31cd9] Grassmann v0.7.6
```
VERSIONINFO
```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Apple M1
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, westmere)
```

**Comment by chakravala** [comment]
Interestingly, this is a bug I just fixed the night before you created this issue. It's now fixed in v0.7.7 in the new commit.

**Comment by umbriquse** [comment]
Haha, well thanks for the timely patch! That worked.
